confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java

```diff
@@ -407,6 +410,35 @@ public TSStatus dropFunction(TDropFunctionReq req) throws TException {
     return configManager.dropFunction(req.getUdfName());
   }
 
+  @Override
+  public TSStatus flush(TFlushReq req) throws TException {
+    if (req.storageGroups != null) {
+      List<PartialPath> noExistSg =
+          configManager.checkStorageGroupExist(PartialPath.fromStringList(req.storageGroups));
+      if (!noExistSg.isEmpty()) {
+        StringBuilder sb = new StringBuilder();
+        noExistSg.forEach(storageGroup -> sb.append(storageGroup.getFullPath()).append(","));
+        return RpcUtils.getStatus(
+            TSStatusCode.STORAGE_GROUP_NOT_EXIST,
+            "storageGroup " + sb.subSequence(0, sb.length() - 1).toString() + " does not exist");
+      }
+    }
+
+    List<TDataNodeInfo> onlineDataNodes;
+    if (req.isLocal) {
```

Comment:
Maybe `isLocal` is unnecessary for `TFlushReq`. It may be confusing if `isLocal` is `false` and `dataNodeId` is not `-1`

Reply:
fixed~